src/node/hooks/express/importexport.js

```diff
@@ -33,13 +38,45 @@ exports.expressCreateServer = function (hook_name, args, cb) {
   });
 
   // handle import requests
+  args.app.use('/p/:pad/import', limiter);
```

Comment:
From now on, Etherpad will always have its import/export rate limited (eventually configurable).

We should mention this in the changelog (I'll do it), and in the documentation.

@JohnMcLear, what is the best place to document the fact that import/export is rate limited?

Reply:
+1 ``settings.json.template`` is good I guess?  Perhaps when a session IS rate limited we should drop something to logs to help admins debug? :)

Reply:
btw +1 good logging, I'm not sure I did this, if you don't mind it should be easy to add as a console.warn with details..  

Ref: onRateLimited: Function called when a request exceeds the configured rate limit.

Perhaps we should include that by default in the blob?   I'm +1 that!

Reply:
Please take care of the documentation part, or at least drop two lines here.

Reply:
See: https://github.com/nfriedly/express-rate-limit#handler for handler controls.

Reply:
Ack, will try sort 2mrw to get this unblocked.  Will take a few days off the list stuff because it's hurting my brain.

Reply:
> it's hurting my brain

You are not alone :)